ckanext/ontario_theme/templates/internal/home/layout3.html

```diff
@@ -4,23 +4,11 @@ <h1 class="sr-only">{{ _("Home") }}</h1>
     <div class="container">
       <div class="hero-container">
         <div class="row">
-          <div class="col-md-6 welcome">
+          <div class="col-md-12 welcome">
             {% block intro %}
               {% snippet 'home/snippets/ontario_theme_intro.html' %}
             {% endblock %}
           </div>
-          <div class="col-md-6 stats">
-            {% block stats %}
-              {% snippet 'home/snippets/ontario_theme_datasets_stats.html' %}
```

Comment:
Since `ontario_theme_datasets_stats.html` is no longer used, you can delete this file from the repo and also delete the references to it in `ckanext-ontario_theme.pot` and `ckanext-ontario_theme.po`.